contribs/gmf/src/services/themesservice.js

```diff
@@ -125,7 +125,7 @@ gmf.Themes.findGroupByLayerName = function(themes, name) {
       var group = theme.children[j];
       for (var k = 0, kk = group.children.length; k < kk; k++) {
         var layer = group.children[k];
-        if (layer.layers == name) {
+        if (layer.name == name) {
```

Comment:
Yes but it's a bit confused. First the name of the function is `findGroupByLayerName` but should be `findGroupByNodeName` to avoid confusion. 
`var layer = group.children[k];` IMO should be `var node = group.children[k];`
In this function, it mixed up the concept of node and layer. This is better to talk about node when manipulating themes and groups and to talk about layer when playing with Openlayers layers.